src/runtime_src/core/tools/common/Escape_Codes.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __ESCAPE_CHAR_H__
+#define __ESCAPE_CHAR_H__
+
+#include <iostream>
+#include <string>
+
+namespace ec {
+#define EC_BLACK 0
+#define EC_RED  1
+#define EC_GREEN  2
+#define EC_YELLOW  3
+#define EC_BLUE 4
+#define EC_MAGENTA  5
+#define EC_CYAN  6
+#define EC_WHITE  7
+#define EC_BRIGHT_BLACK 8
+#define EC_BRIGHT_RED  9
+#define EC_BRIGHT_GREEN  10
+#define EC_BRIGHT_YELLOW  11
+#define EC_BRIGHT_BLUE  12
+#define EC_BRIGHT_MAGENTA  13
+#define EC_BRIGHT_CYAN  14
+#define EC_BRIGHT_WHITE  15
```

Comment:
Replace with constant strings that are scoped to the namespace, and lower case the string names.
But that said, I am not sure where and how these constants are used?
Comments needed.

Reply:
Actually these values are passed as parameters to functions like fgcolour and bgcolour where those functions expects 'int' values. Here I have defined color values that are generally used, but we can pass any value between 0 to 255 to generate different colors, so the parameter should be an integer.

Reply:
Doesn't sound like these should be #defines.  Please change.

Reply:
with the latest commit I have changed them to const uint8_t values, please check in files changed